components/core/src/lib.rs

```diff
@@ -26,6 +26,9 @@ extern crate rustc_serialize;
 extern crate sodiumoxide;
 extern crate libsodium_sys;
 #[cfg(test)]
+#[macro_use]
```

Comment:
My vote would be to change those across all components, as we don't use the single line attribute elsewhere. I'd do this in a separate PR.